src/main/setup/steps/add-content/add-content.ts

```diff
@@ -44,8 +44,11 @@ export const addContentStep: Step = {
       )
     }
 
-    if (styling === "emotion" && inputs.flags.chakra) {
-      promises.push(fs.writeFile("theme.ts", generateTheme()))
+    if (
+      (styling === "emotion" && inputs.flags.chakra) ||
+      inputs.flags["material-ui"]
+    ) {
+      promises.push(fs.writeFile("theme.ts", generateTheme(inputs)))
```

Comment:
I'm tempted to refactor this into an `emotion-theme.ts` and `mui-theme.ts`, so we won't get trouble with people wanting to use both. It looks like you have Chakra UI taking precedence at the moment.

For people using only one component library, it would be better UX with just `theme.ts`, but I think that UX benefit is pretty small compared to being able to add two component libraries.

It would also be possible to add a check to see if only one component library was chosen. At that point, I think it would make sense if `add-content.ts` calls an `addThemes` function where the choice of the theme file name(s) is colocated with the actual theme generation.

I'm unsure, let me know if you have thoughts.

Reply:
I can have a look into that, in the `generateTheme` function, it could return an array with the filename and actual theme data so we could in theory just pass that straight to the current `fs.writeFile` function and inside the `generateTheme` function we could run those checks to figure out the file name and what-not

Reply:
I had another idea, which would probably work better than the one mentioned above

When the `generateTheme` function is called it checks whether Chakra, Material UI or both are checked, and based on the results it returns, between 0 and 2 `fs.writeFile` functions which can then be passed into `promises.push`

Reply:
If you wanna take a look at [this commit](https://github.com/jamesinaxx/create-next-stack/commit/a56f454d67ea9cfd721bee4598306c14673994cb), it's the idea I mentioned above

Reply:
This is great James, I like it! 😊